css/index.css

```
@@ -0,0 +1,309 @@
+body {
+  background-color: #f2f6f9;
+  margin: 0;
+  font-family: "Poppins";
+  font-weight: 500;
+}
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  background-color: #f2f6f9;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.btn-back {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 4px 8px;
+  border: 1px solid #081c3933;
+  border-radius: 5px;
+  width: 73px;
+  height: 32px;
+  font-family: "Poppins", sans-serif;
+  font-size: 14px;
+  background-color: #f2f6f9;
+  position: absolute;
+  top: 32px;
+  left: 32px;
+}
+
+.img {
+  width: 16px;
+  height: 24px;
+  margin-right: 6px;
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-top: 28px;
+  font-weight: 600;
+  font-size: 48px;
+  line-height: 72px;
+  margin-bottom: 32px;
+  color: #0C2447;
+}
+
+.subtitle {
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #203a62;
+}
+
+.switch-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-top: 40px;
+}
+
+.monthly {
+  font-size: 16px;
+  line-height: 24px;
+  color: #203a62;
+  margin-right: 12px;
+}
+
+.img-switch {
+  margin-right: 12px;
+}
+
+.annually {
+  font-size: 16px;
+  line-height: 24px;
+  color: #6f809a;
+  margin-right: 12px;
+}
+
+.save {
+  font-size: 14px;
+  line-height: 21px;
+  color: #74ca77;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 32px auto 0 auto;
+  column-gap: 20px;
+  row-gap: 50px;
+  flex-wrap: wrap;
+  padding-bottom: 80px;
+  box-sizing: border-box;
+}
+
+.plan {
+  width: 320px;
+  height: 593px;
```

no need to make pixsel-prefect + min height (more save)
```suggestion
   min-height: 600px;
```